### ISSUE TITLE
Profile: Sprint 1 (1.4.23 - 1.17.23)

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#51107]
## PTO/availability

- Federal government closed 1/16

## Goals

- Welcome new team members!
- Continue accessibility work.
- Kick off direct deposit migration work.
- Kick off notification preferences research around adding email as a channel.

## General

**Onboarding**

- [x] #50964
- [x] #50956

**FE - Adam**

- [ ] #50889 (merged and waiting for staging deployment)

## Accessibility updates

**FE - Adam**

- [ ] #49651 (in PR, and should be deployed once reviewed)
- [x] #49374
- [ ] #49550 (didn't get to this ticket)

## Direct deposit for comp & pen: Lighthouse migration

**BE - Tom**

- [x] #51103

## Notification preferences

**Design - Liz**

- [x] #51007
- [x] #50999
- [x] #51000
- [x] #51001
- [x] #51002 

## Unplanned work

**IA research - Liz**

- [x] #51750
